app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -67,7 +67,7 @@ keywords:
 
 Citizens of every country in the world can apply to train to teach in England. When you plan your training, bear in mind that:
 
-- international candidates can expect to pay an average of £14,765 to train, plus
+- international candidates can expect to pay an average of $fees_pgitt_internationalaverage$ to train, plus
```

Comment:
This is £15,000 on the review app and £14,765 on the live site, I think this is right as £14,765 is a weird average, but wanted to flag just in case

Reply:
Request from ITRU team to update